app/services/check_cmc_ohclv_service.rb

```diff
@@ -71,23 +72,24 @@ def call
   def check_coins
     @coin_tests.each do |coin_test|
       # Check each individual coin within a test
-      coins = coin_test[:query].call
+      coins = coin_test[:query].call.to_a
 
       if coins.length == 0
         @failed_tests << coin_test[:title]
         next
       end
 
       coins.each do |coin|
-        coin_key = coin[0].to_s
-        ranking = coin[1]
+        coin_key = coin.coin_key.to_s
```

Comment:
Just curious, when is a `coin_key` not a string?

Reply:
I honestly can't remember why I had that there. It may have just been because I was performing string operations and wanted to make sure it was a string? It can probably be removed but I didn't want to break anything haha.